### PR TITLE
hotfix: go bindings

### PR DESCRIPTION
**Pull request by arsen3d**
### Review Type Requested (choose one):
- [ ] **Logic** - thorough check (from everybody doing review)

### Summary
Calling ./stack go-bindings after compile

### CODE REVIEW

File: docker/chain/Dockerfile [hunk 1]
```diff
@@ -55,19 +55,23 @@ RUN echo "npx hardhat deploy --network devnet" >> run
 RUN echo "npx hardhat run scripts/fund-services-ether.ts  --network devnet" >> run
 RUN echo "npx hardhat run scripts/fund-services-tokens.ts  --network devnet " >> run
 RUN echo "npx hardhat run scripts/print-l2-config.ts  --network devnet" >> run
+RUN echo "cd .." >> run
+RUN echo "./stack go-bindings" >> run
```
Comment (AquiGorka) [inline docker/chain/Dockerfile:59]: Why not run `chain-boot` directly?

File: docker/chain/Dockerfile [hunk 2]
```diff
@@ -55,19 +55,23 @@ RUN echo "npx hardhat deploy --network devnet" >> run
...
+
+
 
 FROM base AS expose-cloudflare
 ARG cloudflare_token_http="not-a-token"
 ARG cloudflare_token_ws="not-a-token"
 
 RUN wget https://github.com/cloudflare/cloudflared/releases/latest/download/cloudflared-linux-amd64
-RUN echo "cd .." >> run
+# RUN echo "cd .." >> run
```
Comment (AquiGorka) [inline docker/chain/Dockerfile]: Let's remove this instead of commenting it out

File: docker/chain/Dockerfile [hunk 3]
```diff
@@ -55,19 +55,23 @@ RUN echo "npx hardhat deploy --network devnet" >> run
...
 RUN echo "./cloudflared-linux-amd64 tunnel run --token $cloudflare_token_http  &" >> run
 RUN echo "./cloudflared-linux-amd64 tunnel run --token $cloudflare_token_ws &" >> run
 RUN chmod +x cloudflared-linux-amd64
 
 FROM base AS expose-local
-RUN echo "cd .." >> run
+# RUN echo "cd .." >> run
```
Comment (AquiGorka) [inline docker/chain/Dockerfile]: Same as above, let's remove this instead of commenting it out

File: docker/chain/Dockerfile (hunk 1 above)
Comment (arsen3d) [reply docker/chain/Dockerfile:59]: --network local_l2 vs  --network devnet

**Review by AquiGorka — APPROVED** [review]
⛓️ ⛓️ thanks @arsen3d !

**Review by richardbremner — APPROVED** [review]

**Pull request merged**